src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -640,12 +667,14 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
                             />
                         </div>
                         <div>
-                            <ProtectedInput
+                            <CustomPassInput
                                 value={this.state.form.token}
                                 onChange={(event) => this.handleChange(event, 'token')}
                                 name="Enter token"
                                 tabIndex={4}
                                 error={this.state.isError.token}
+                                onBlur={this.handleOnBlur}
```

Comment:
Using these two (onblur/onfocus) in multiple places. Please abstract this out in common place